Makefile

```diff
@@ -18,7 +18,7 @@ test4:
 devdeps:
 	pip install --upgrade pip setuptools
 	pip install wheel twine
-	pip install 'black==21.12b0' 'pytest>=6.0' pytest-cov pytest-xdist pytest-sugar myst-parser sphinx sphinx-argparse
+	pip install 'black==22.3' 'pytest>=6.0' pytest-cov pytest-xdist pytest-sugar myst-parser sphinx sphinx-argparse
```

Comment:
Incidental change, related to the issue described [here](https://github.com/psf/black/issues/2964). Popped up recently and started causing CI build failures.